src/home/App.js

```diff
@@ -84,19 +50,21 @@ class App extends Component {
                 to={{ opacity: 1, x: 0 }}
                 keys={appIcons.map(item => item.key)}
               >
-                {appIcons.map((appIcon, i) => ({ x, opacity }) => (
+                {appIconsWithReadOnlyInfo.map((appIcon, i) => ({ x, opacity }) => (
```

Comment:
In general in map(), if we don't use the `i` parameter then it should be omitted, imo.

Reply:
done